yotacli/src/components/batch/createbatch/BatchForm.js

```diff
@@ -130,6 +130,7 @@ const BatchForm = (props) => {
         alert("Batch created sucessfully...")
         e.preventDefault();
         batches['batchName'] = calculateBatchName();
+        batches['batchIdentifier'] = "Pankaj Sharma";
```

Comment:
Please add username instead of batchIdentifier, becuase we are calculating identifier based on user name in backend.

Reply:
resolved